app/questionnaire/questionnaire_schema.py

```diff
@@ -81,6 +82,7 @@ def __init__(
         ] = defaultdict(set)
         self._language_code = language_code
         self._questionnaire_json = questionnaire_json
+        self.min_and_max_map: dict[str, str] = {}
```

Comment:
Might be more consistent with the other attributes if this was private and there was a cached property that returned an immutable version of the map